docs/4.tutorials-and-examples/2.human-handoff.mdx

```diff
@@ -65,5 +63,5 @@ export default class extends React.Component {
 ```
 
 > #### NOTE:
-> Remember that `getOpenQueues` and `humanHandOff` are only available when the bot is deployed!
+> Remember that the full features for `getOpenQueues` and `humanHandOff` are only available when the bot is deployed!
```

Comment:
when you mention full features do you mean that it works partially with "botonic serve"?  If not, I would like it as it was

Reply:
yes, this is what I would say.